.github/pull_request_template.md

```diff
@@ -1,9 +1,15 @@
-# Related Issues
-<!-- Issue(s) that this PR will resolve, e.g. "resolves <issue link here>". -->
-
-# Summary of Changes
+## Summary of Changes
 <!-- A summary or description of changes made in this PR. -->
 
-# Steps to Reproduce
+## Motivation and Explanation
+<!-- How and why do your changes improve the bot? -->
```

Comment:
Add a section for screenshots/video showcasing the new feature.

Reply:
That's covered in the `Steps to Reproduce` section below isn't it?

Reply:
Make it an explicit heading - people can remove it or say "n/a" if there are no screenshots to show.